### DISCUSSION TITLE
Run tests in release mode

### DISCUSSION THREAD

**Discussion by kryptan** [rust-lang/rust-analyzer#15519] [Q&A]
Is it possible to configure vscode to run tests in release mode (`cargo test --release`) when I press the `Run Test` button? I'm primarily using release mode and running tests causes rebuilding all of the dependencies and I want to avoid that.

![image](https://github.com/rust-lang/rust-analyzer/assets/11528855/aa42e106-5d43-4590-a9c1-219cca06b062)

**Comment by lnicola** [comment]
https://github.com/rust-lang/rust-analyzer/pull/11730 is somewhat related, but no.
